src/babel/generation/generators/expressions.js

```diff
@@ -133,6 +133,12 @@ export function AssignmentExpression(node, print) {
   print(node.right);
 }
 
+export function BindExpression(node, print) {
```

Comment:
Actually, there's no way this function could possibly be hit... If the plugin is off then it's a parse error if it's on then it's transformed.